zcash_primitives/src/legacy/keys.rs

```diff
@@ -21,7 +21,7 @@ use super::TransparentAddress;
 pub struct TransparentKeyScope(u32);
 
 impl TransparentKeyScope {
-    pub fn custom(i: u32) -> Option<Self> {
+    pub const fn custom(i: u32) -> Option<Self> {
```

Comment:
It is definitely a false positive that this isn't covered by tests. It's executed to construct `EPHEMERAL_SCOPE`.